test/UnitTest/Altinn.Platform.Storage.UnitTest.csproj

```diff
@@ -161,4 +162,16 @@
     </None>
   </ItemGroup>
 
+	<ItemGroup>
+		<None Update="JWTValidationCert.cer">
```

Comment:
Do you need this validation certificate or are the platform-org.*

Reply:
Seems it's not a part of the commit, so only the property that needs removing from this file :)

Reply:
Manual project file changes, always fun. Fixed now.